QueryBuilder/Compilers/Compiler.cs

```diff
@@ -834,6 +834,8 @@ public virtual string WrapValue(string value)
             var opening = this.OpeningIdentifier;
             var closing = this.ClosingIdentifier;
 
+            if (string.IsNullOrWhiteSpace(opening) && string.IsNullOrWhiteSpace(closing)) return value;
```

Comment:
Doesn't this fail when passing an empty string for the opening only?

Reply:
Previous behaviour would not test for inconsistent `opening ` and  `closing` values.
Looks like this fix preserves that behaviour.